src/store/i18n/store.ts

```diff
@@ -94,17 +95,26 @@ export const useI18nStore = create<I18nState & I18nActions>()((set) => ({
 						newI18n
 							// load translation using http -> see /public/locales (i.e. https://github.com/i18next/react-i18next/tree/master/example/react/public/locales)
 							// learn more: https://github.com/i18next/i18next-http-backend
-							.use(Backend)
+							.use(ChainedBackend)
 							// init i18next
 							// for all options read: https://www.i18next.com/overview/configuration-options
 							.init({
 								...defaultI18nInitOptions,
 								lng: locale,
 								backend: {
-									loadPath:
-										app.name === SHELL_APP_ID
-											? `${BASE_PATH}/i18n/{{lng}}.json`
-											: `${dropRight(app.js_entrypoint.split('/')).join('/')}/i18n/{{lng}}.json`
+									backends: [HttpBackend, HttpBackend],
```

Comment:
2 times HttpBackend?

Reply:
Yes, the second one is the fallback which will use the second configuration of the backendOptions field

Reply:
Yes, the second one is the fallback which will use the second configuration of the backendOptions field